crates/miden-objects/Cargo.toml

```diff
@@ -34,13 +34,14 @@ testing = ["dep:rand", "dep:rand_xoshiro", "dep:winter-rand-utils"]
 
 [dependencies]
 # Miden dependencies
-miden-assembly    = { workspace = true }
-miden-core        = { workspace = true }
-miden-crypto      = { workspace = true }
-miden-processor   = { workspace = true }
-miden-utils-sync  = { workspace = true }
-miden-verifier    = { workspace = true }
-winter-rand-utils = { optional = true, version = "0.13" }
+miden-assembly     = { workspace = true }
+miden-core         = { workspace = true }
+miden-crypto       = { workspace = true }
+miden-mast-package = { default-features = false, version = "0.17.0" }
```

Comment:
nit: I'd probably put this at the workspace level and then user here as `{ workspace = true }`.

Also, I'd change `0.17.0` to just `0.17`.